components/BottomNavBar.tsx

```diff
@@ -0,0 +1,66 @@
+import Link from 'next/link';
+import DashboardIcon from './icons/DashboardIcon';
+import SearchIcon from './icons/SearchIcon';
+import SiteListIcon from './icons/SiteListIcon';
+import { useRouter } from 'next/router';
+import useSWR, { Fetcher } from 'swr';
+import { SiteListProps } from '~/lib/types';
+
+const icons = [
+  {
+    title: 'Dashboard',
+    icon: DashboardIcon,
+    link: '/dashboard',
+  },
+  {
+    title: 'My Sites',
+    icon: SiteListIcon,
+    link: '/sites',
+  },
+  {
+    title: 'More Info',
+    icon: SearchIcon,
+    link: '/more-info',
+  },
+];
+
+const BottomNavBar = () => {
+  const { asPath } = useRouter();
+  const fetcher: Fetcher<SiteListProps> = async (url: string) => {
+    return await fetch(url).then((res) => res.json());
+  };
+  const { data, isLoading } = useSWR(
+    `${process.env.NEXT_PUBLIC_API_BASE_URL}/api/sites/site_list`,
+    fetcher
+  );
+
+  return (
+    <div
+      className={`${
+        data?.site_list.length === 0
+          ? 'opacity-0 pointer-events-none hidden'
+          : 'opacity-100'
+      }
+    bg-ocf-gray-1000 w-screen h-[80px] bottom-0 fixed visible md:invisible`}
+    >
+      <div className="flex justify-evenly mt-[15px]">
```

Comment:
lets change the height of the bottom navbar here-a height of 60px looks reasonable to me, and the margin-top of the child div needs to change as well to make it look less wonky. 

```suggestion
    bg-ocf-gray-1000 w-screen h-[60px] bottom-0 fixed visible md:invisible`}
    >
      <div className="flex justify-evenly mt-[5px]">
```